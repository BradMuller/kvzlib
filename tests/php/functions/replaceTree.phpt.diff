001- bool(true)
002+ Fatal error: Call to undefined function replaceTree() in /home/kevin/workspace/plutonia-kvzlib/php/functions/replaceTree.phpt.php on line 10