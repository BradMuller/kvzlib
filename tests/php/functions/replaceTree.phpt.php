<?php

require_once str_replace(".example.", ".inc.", __FILE__);

$settings = array(
    "Credits" => "@appname@ created by @author@",
    "Description" => "@appname@ can parse logfiles and store then in mysql"    
);    

$settings = replaceTree("@appname@", "logchop", $settings);
var_dump($settings);

?>