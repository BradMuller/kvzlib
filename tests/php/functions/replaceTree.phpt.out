
Fatal error: Call to undefined function replaceTree() in /home/kevin/workspace/plutonia-kvzlib/php/functions/replaceTree.phpt.php on line 10
